test/resource/resource.go

```diff
@@ -12,7 +12,11 @@ type FakeResource struct {
 	CustomMap map[string]struct {
 		Tag string
 	}
-	Slice []string
+	Slice     []string
+	SubStruct struct {
```

Comment:
It will conflict with @wbeuil PR https://github.com/cloudskiff/driftctl/pull/29/files#diff-3c37a382b906e1868966fe82aee93c8b2d822345546f47de108ed447a760c494R16

We need to choose the same nesting struct for FakeResources

Reply:
Yeah I just need a nested struct but there was not before. I will change it on rebase

Reply:
You could change it now to the same struct as @wbeuil, so the rebase will be straightforward without conflicts

Reply:
Oh nice catch on this one ! I needed a struct just to test for nested computed fields.